homeassistant/components/switcher_kis/button.py

```diff
@@ -85,9 +90,10 @@ async def async_setup_entry(
 
     async def async_add_buttons(coordinator: SwitcherDataUpdateCoordinator) -> None:
         """Get remote and add button from Switcher device."""
+        data = cast(SwitcherBreezeRemote, coordinator.data)
```

Comment:
Not sure I like the use of `cast` here. Maybe there is a better solution, but that can be explored later.